tests/tests.py

```
@@ -2,40 +1,272 @@
+        # Simply create the provider with the settings
+        return provider_cls(settings)
 
-class TestStorageStore(TestStorageBase):
+    def teardown_method(self, method):
+        """Clean up after tests."""
+        # Make sure connections are closed properly
+        try:
+            provider = self.get_storage_provider()
+            if hasattr(provider, "cleanup"):
+                provider.cleanup()
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)